tests/baselines/reference/promisePermutations.errors.txt

```diff
@@ -25,8 +25,6 @@ tests/cases/compiler/promisePermutations.ts(106,19): error TS2345: Argument of t
   Types of parameters 'cb' and 'value' are incompatible.
     Type 'string' is not assignable to type '<T>(a: T) => T'.
 tests/cases/compiler/promisePermutations.ts(109,19): error TS2345: Argument of type '(cb: <T>(a: T) => T) => IPromise<string>' is not assignable to parameter of type '(value: string) => IPromise<string>'.
-  Types of parameters 'cb' and 'value' are incompatible.

```

Comment:
We have already shown it in a previous elaboration. The reason it is going away is that we do a better job of sharing identical types, so you'll see less redundant elaborations. This is the expected behavior.